app/Console/Commands/CleanUpMastodonServers.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\MastodonServer;
+use App\Models\User;
+use App\Notifications\InvalidMastodonServer;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+
+class CleanUpMastodonServers extends Command
+{
+    protected $signature = 'trwl:clean-up-mastodon-servers';
+
+    protected $description = 'Clean up Mastodon servers...';
+    private const string DIVIDER       = '====================';
+    private const string SMALL_DIVIDER = '-------------------';
+
+
+    public function handle(): void {
```

Comment:
Idea: In order to be able to track the changes made here, I would log the actions so that we can see this in the admin panel.

See here for easy examples: https://spatie.be/docs/laravel-activitylog/v4/basic-usage/logging-activity

Reply:
I might add this down the road. Or feel free to add it yourself. :)